### PR TITLE
DOC-000: add github action badge to readme

### PR DESCRIPTION
**Pull request by phong1233**
# Description

Added github action badge to readme
moved firebase deployment to frontend ci/cd

## Type of change

Please delete options that are not relevant.

- [ ] BUG (Fixing or documenting a bug)
- [ ] FEAT (Add a new feature to the project)
- [ ] CHORE (Something that needs to be done outside of the issue)
- [x] DOCS (Implement documentation)
- [ ] TASK (A task that must be done to complete an issue)

# Checklist:

- [x] My pull request references the issue in the comments
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sebastien-blain — APPROVED** [review]
Hop firebase works with this

**Comment by phong1233** [comment]
> Hop firebase works with this

I don't have high hopes

**Review by Talos10 — APPROVED** [review]
Great job with the firebase hosting setup!

File: frontend/src/utils/api.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const api = process.env.REACT_APP_API ?? 'http://localhost:5000';
+const api = process.env.REACT_APP_API ?? 'https://supreme-erp.herokuapp.com/';
```
Comment (sebastien-blain) [inline frontend/src/utils/api.tsx]:
```suggestion
const api = process.env.REACT_APP_API ?? 'https://supreme-erp.herokuapp.com';
```

**Review by sebastien-blain — APPROVED** [review]

**Pull request merged**